block_my_feedback.php

```diff
@@ -212,7 +212,8 @@ public function is_course_current(stdClass $course): bool {
             if ($course->enddate == 0) {
                 return true; // Enddate is set to 0 when no end date, show course.
             }
-            if (time() > $course->enddate) {
+            $future = strtotime('+3 month'); // Allow for Late summer assessments.
+            if ($future > $course->enddate) {
```

Comment:
Are we sure about this logic? If the course end date is less than three months away don't show it? Seems a bit odd to me, but then I don't know what Late Summer Assessments are, nor what bearing they have on course end dates. Maybe the comment could explain a little more?

Reply:
I'd also spotted in testing...